MapboxNavigation.podspec

```diff
@@ -45,7 +45,6 @@ Pod::Spec.new do |s|
 
   s.dependency "MapboxCoreNavigation", "#{s.version.to_s}"
   s.dependency "Mapbox-iOS-SDK", "~> 3.6"
-  s.dependency "Pulley", "1.4"
```

Comment:
One last to remove: https://github.com/mapbox/mapbox-navigation-ios/blob/master/MapboxNavigation-Documentation.podspec#L49